weather_dl/download_pipeline/pipeline_test.py

```diff
@@ -632,5 +635,19 @@ def test_target_name_date_and_target_directory_additional_partitions_in_path(sel
         self.assertEqual(target_name, "somewhere/expver-1/2017/01/15-pressure-500.nc")
 
 
+class ConfigTest(unittest.TestCase):
```

Comment:
Let's move these config tests to a separate test file. To me, they are independent from the pipeline tests.